packages/ripple-nuxt-tide/lib/components/AppForm.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <rpl-form v-if="formData" :formData="formData" :submitHandler="submitForm" :hideAfterSuccess="formData.settings.shouldHideFormAfterSuccess" :title="title" ></rpl-form>
+  <rpl-form v-if="formData" novalidate :name="`form-${formData.tideId}`" :formData="formData" :submitHandler="submitForm" :hideAfterSuccess="formData.settings.shouldHideFormAfterSuccess" :title="title" ></rpl-form>
```

Comment:
Hey @dylankelly Is there a particular reason to use `novalidate` on webform?

Reply:
yes, the html validation is inconsistent with VFG validation, flaky tests identified potentially buggy code.